### ISSUE TITLE
/notes can dominate crawl

### ISSUE THREAD

**Issue by marked** [ArchiveTeam/tumblr-grab#10]
http://squirrel.tumblr.com/
9 posts, but >1million notes
```
Bytes RootDir  -------------------------------------------
               images :   2 % =    2749514 /  151998096
               client :   2 % =    3399437 /  151998096
                fonts :   5 % =    7299194 /  151998096
                notes :  63 % =   96041196 /  151998096
                 post :   2 % =    2868475 /  151998096
                  pop :  17 % =   26088561 /  151998096

Hits RootDir  -------------------------------------------
               images :   4 % =        312 /       7848
                fonts :   3 % =        236 /       7848
                notes :  76 % =       5948 /       7848
              archive :   1 % =         80 /       7848
                 post :   1 % =         98 /       7848
                  pop :  10 % =        746 /       7848
```

**Comment by marked** [comment]
Leading proposal is take the 1 page of 'notes' but don't paginate to the others.
Reasoning it let's the pane have some content but without interrupting crawl speed.

Unclear if URL stream lets us do this via string matching.

**Comment by nemobis** [comment]
What's wrong with notes? They seem a way to keep curation data useful to make sense of the whole mirror down the road, with low storage cost (compared to posts which duplicate media).

**Comment by nightpool** [comment]
@nemobis They lengthen archiving jobs drastically, and are mostly duplicate content. Because "notes seen" scales with square of reblogs, it quickly becomes impossible to store them effectively. For example, on a "small" post with 10,000 reblogs, and 50 notes per page, it becomes necessary to crawl *2,000,000* notes pages. 

If we could figure out a way to only crawl notes on *root* posts, then that would solve the problem, but that seems like a lot of logic for this small bot.

**Comment by marked** [comment]
Consensus is to crawl only first post of /notes.
Someone needs to dive into a .warc and post back the URLS for the sequence.

**Comment by kiska3** [comment]
I am proposing this as a fix to this issue: https://github.com/ArchiveTeam/tumblr-grab/commit/87d69147f422867dd353d73ab091cd515597ade2

**Comment by marked** [comment]
I proposed patch-8 or patch-9 as a fix.
Testing, these did not pass.

However, adding /notes to the block list did prevent the long notes chains.
Unexpectedly it did NOT affect the 1st page of notes.  It seems that page is prefilled in by the appserver.

**Comment by marked** [comment]
Fix in PR #34 
notes section is prefilled in so the /notes URI's are for extra notes we're deciding to skip for crawl speed.
1 set is enough.
